src/types/modules/electron.d.ts

```diff
@@ -0,0 +1,9 @@
+declare global {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+    interface Window {
+        electron: Electron.IpcRenderer;
```

Comment:
NAB: This type might be too extensive, we should probably create a type based on the implementation [here](https://github.com/software-mansion-labs/expensify-app-fork/blob/fd4fa9f6d4a4ecaeb437589d153495782e29611c/desktop/contextBridge.js). [Article](https://www.jsgarden.co/blog/how-to-handle-electron-ipc-events-with-typescript) that explains it well.